src/io/gff.rs

```diff
@@ -53,6 +54,24 @@ pub enum GffType {
     Any(u8, u8, u8),
 }
 
+impl FromStr for GffType {
+    type Err = ();
+
+    /// Create a GffType from a string.
+    ///
+    /// # Arguments
+    ///
+    /// * `src_str` - The source string to convert to the GffType.
+    fn from_str(src_str: &str) -> Result<Self, Self::Err> {
+        match src_str {
+            "gff3" => Ok(GffType::GFF3),
+            "gff2" => Ok(GffType::GFF2),
+            "gtf2" => Ok(GffType::GTF2),
+            _ => Err(()),
```

Comment:
Maybe add in an error message, here? E.g. `Err(format!("String {} is not a valid GFFType (GFF/GTF format version).", src_str) )`?

Reply:
037b8b5